index.Rmd

```diff
@@ -1,54 +1,107 @@
 
-You can do numbers like this:
+#### Analysisng MODIS fire data 
 
-1. The first most important thing
-2. The second most important thing
-3. The third most important thing
+The MODIS fire data are in the form of csv files, once these files are read into R, they are in the form of data frames, the number of times a brightness value occurs in a day is taken as a fire count for that day. Number of fire counts on each day are obtained. Based on these fire counts I have filtered the study dates and would be further focusing on them. 
 
-See [http://rmarkdown.rstudio.com/](http://rmarkdown.rstudio.com/) for all the amazing things you can do.
+#### Analysing GEOS-CF model data for NO~2~  and CO 
 
+GEOS-CF model data are in the form of .h5 files, the data in these files is in the form of a meshgrid/ matrix. Longitude and Latitude values  are distributed in the form of a grid, and each point in the grid (longitude,latitude) pair has a concentration value. 
+GEOS-CF model data on specific dates is chosen based on the MODIS data, the days on which MODIS fire counts were found to be significant were taken into consideration. 
+
+#### Data Representation 
+
+Data for TROPOMI CO and NO~2 are represented in the from of maps with longitudes varying on x-axis and latitudes varying on y-axis. GEOS-CF model data is also represented in the same way. 
+
+The following packages were used in the project:
+
+```{r, message=F, warning=F}
+library(pracma)
 
-Here's my first code chunk.
-```{r}
-1+2
 ```
 
-Load any required packages in a code chunk (you may need to install some packages):
 
 ```{r, message=F, warning=F}
+library(dplyr)
+library(R.matlab)
+library(lubridate)
+library(ncdf4)
+library(ggplot2)
+library(devtools)
+library(Bilinear)
+library(interp)  
+```
+
+
+```{r, message=F, warning=F}
+library(R.matlab)
 library(tidyverse)
 library(leaflet)
 library(kableExtra)
-knitr::opts_chunk$set(cache=TRUE)  # cache the results for quick compiling
+knitr::opts_chunk$set(cache=TRUE) 
 ```
 
+# DATA HANDLING 
+#### Reading modis fire data 
+```{r , results='asis'}
+
+modis_data <- read.csv('C:\\Users\\shrut\\Desktop\\SDS Project\\fire_archive_M6_164945.csv')
+
+```
+
+####  MODIS DATA: Setting up Latitude, Longitude boundaries and sorting data 
+```{r , results='asis'}
+
+modis_data_lat_filter <- modis_data[modis_data$latitude > 30.4 & modis_data$latitude < 37.3 ,]
+modis_data_lon_filter <- modis_data_lat_filter[modis_data_lat_filter$longitude > -124 & modis_data_lat_filter$longitude < -111.8 ,]
+dates <- c(modis_data_lon_filter['acq_date'])
+count_table <- as.data.frame(table(dates))
+counts_table_order <- count_table[order(-count_table$Freq),]
+```
+
+From the above code we can have a clear view about the dates where fire was significant, the table counts_table shows the frequency of fire counts on each day. 
+
+#### Reading GEOS-CF NO~2~ DATA
+
+```{r , results='asis'}
+setwd('C:\\Users\\shrut\\Downloads')
```

Comment:
Don't change the working directory in the script.  Move the file into the repository in the /data directory and then load it from there.